docs/DeveloperGuide.md

```diff
@@ -882,41 +1193,40 @@ The category is a single filter that is matched in a case-insensitive manner
 
 #### Delete Transaction
 
-##### Overview
+**Overview**
 
-The `DeleteCommand` function deletes an existing `Transaction` from `TransactionList` in UniCash.
+The `delete_transaction` command deletes an existing `Transaction` from the `TransactionList` in UniCa$h.
 
 The activity diagram of deleting a Transaction is as shown below
 
-<img src="images/unicash/DeleteTransactionActivityDiagram.png" width="600" />
+<img src="images/unicash/diagrams/DeleteTransactionActivityDiagram.png" width="500" />
 
 The following sequence diagram shows the interaction between different components of UniCash.
 
-<img src="images/unicash/DeleteTransactionSequenceDiagram.png" width="1200" />
+<img src="images/unicash/diagrams/DeleteTransactionSequenceDiagram.png" width="1200" />
 
 The above sequence diagram omits details on the filtering of `TransactionList` and assumes that
 the displayed `TransactionList` is showing all transactions. However, the logic of the `DeleteCommand`
 remains the same for all list deletion.
 
-ℹ️ **Note:** The lifeline for `DeleteTransactionCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML,
-the lifeline reaches the end of diagram.
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+**Note:** The lifeline for `DeleteTransactionCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+</div>
 
-##### Details
+**Details**
```

Comment:
Hmmm... Is there a reason why u changed this to ** instead of ##### for the "Details" section? I believe other commands also do it in the ##### format!😄